layers/+lang/ocaml/funcs.el

```diff
@@ -23,26 +23,30 @@
 
 (defun spacemacs//init-ocaml-opam ()
   (if (executable-find "opam")
-      (let ((share (string-trim-right
-                    (with-output-to-string
-                      (with-current-buffer
-                          standard-output
-                        (process-file
-                         shell-file-name nil '(t nil) nil shell-command-switch
-                         "opam config var share"))))))
-        (cond ((string= "" share)
-               (spacemacs-buffer/warning
-                "\"opam config var share\" output empty string."))
-              ((not (file-directory-p share))
-               (spacemacs-buffer/warning
-                "opam share directory does not exist."))
-              (t (setq opam-share share
-                       opam-load-path (concat share "/emacs/site-lisp"))
-                 (add-to-list 'load-path opam-load-path))))
+        (let* ((share-dir-from-command (lambda (command)
+                (string-trim-right (with-output-to-string (with-current-buffer standard-output
```

Comment:
the indentation is still not right.
try to press `= =` on the line `(process-file shell-file-name`.

Reply:
for indentation, either write
```elisp
(foo bar
     baz
     xyz)
```
or
```elisp
(foo
 bar
 baz
 xyz)
```

but never
```elisp
(foo bar
 baz
 xyz)
```